tools/wltests/test_series

```diff
@@ -118,7 +124,9 @@ Additional arguments:
 	--adb			The ADB binary to use.
 				Default: ADB=[ANDROID_HOME/platform-tools/adb|\$(which adb)]
 	--fastboot		The FASTBOOT binary to use.
-				Default: FASTBOOT=[ANDROID_HOME/platform-tools/fatboot|\$(which fatboot)]
+				Default: FASTBOOT=[ANDROID_HOME/platform-tools/fastboot|\$(which fastboot)]
+	--repo			The REPO binary to use.
```

Comment:
Does it make sense to override `repo`? It's a bit like specifying which `git` binary to use, no?

Reply:
Right, perhaps I can remove that one :)